artiwrapper/src/main/java/org/c4dt/artiwrapper/Client.java

```diff
@@ -36,6 +38,37 @@ public enum TorRequestMethod {
         }
     }
 
+    /**
+     * Check that all required files are present in the cache directory.
+     *
+     * @param cacheDir cache directory path
+     */
+    static String checkCacheDirFiles(String cacheDir) throws TorLibException {
```

Comment:
This code needs to be part of the `lightarti-rest`, as it's used by both the iOS and the android client.

Do you feel comfortable implementing this in there? The best place would be in the `TorClientConfig::builder`, but this would require modification of the `arti-client` - which would be nice and also useful for others.

The second-best place is here:

https://github.com/c4dt/lightarti-rest/blob/main/src/client.rs#L40

We can discuss this tomorrow, if you want. This afternoon I'm with SICPA.

Reply:
OK, I'll move it